webapi/src/main/scala/org/knora/webapi/messages/v2/responder/valuemessages/ValueMessagesV2.scala

```diff
@@ -3420,6 +3491,44 @@ object DocumentFileValueContentV2 extends ValueContentReaderV2[DocumentFileValue
   }
 }
 
+/**
+ * Constructs [[BundleFileValueContentV2]] objects based on JSON-LD input.
+ */
+object BundleFileValueContentV2 extends ValueContentReaderV2[BundleFileValueContentV2] {
+  override def fromJsonLDObject(
+    jsonLDObject: JsonLDObject,
+    requestingUser: UserADM,
+    responderManager: ActorRef,
+    storeManager: ActorRef,
+    featureFactoryConfig: FeatureFactoryConfig,
+    settings: KnoraSettingsImpl,
+    log: LoggingAdapter
+  )(implicit timeout: Timeout, executionContext: ExecutionContext): Future[BundleFileValueContentV2] = {
+    implicit val stringFormatter: StringFormatter = StringFormatter.getGeneralInstance
+
+    for {
+      fileValueWithSipiMetadata <- FileValueWithSipiMetadata.fromJsonLDObject(
+        jsonLDObject = jsonLDObject,
+        requestingUser = requestingUser,
+        responderManager = responderManager,
+        storeManager = storeManager,
+        settings = settings,
+        log = log
```

Comment:
If `name = name` you can probably replace it just with `name`.

Reply:
as above